### DISCUSSION TITLE
path for mini  labs 11 2.00.70 - Jan 5, 2023 update?

### DISCUSSION THREAD

**Discussion by antonimurcia** [gopro/labs#179] [General]
hello!
the gopro labs FW update page for the gopro mini still points to the  v1.10.70 | Nov 2022 FW update, but the release notes 
hint on a 2.00.70 - Jan 5, 2023 update 
Is the update ready? where can we find it?

thanks in advance

**Comment by dnewman-gpsw** [comment]
It was released, just waiting for the website to update.   

Here it is in the meantime https://gopro.github.io/labs/firmware/bin/LABS_MINI11_02_00_70.zip

**Reply by antonimurcia** [reply]
perfect, many thanks!!!!!!

**Reply by LukeRuby83** [reply]
Thanks!
